test_aodncore/pipeline/test_dummyHandler.py

```diff
@@ -43,14 +44,17 @@ def test_dest_path_from_string(self):
         self.assertIs(handler._dest_path_function_ref, dest_path_testing)
 
     def test_include(self):
-        handler = self.run_handler(BAD_ZIP, include_regexes=['good\.nc'])
-        eligible_filenames = [f.name for f in handler.file_collection if f.should_harvest]
+        handler = self.run_handler_with_exception(UnmatchedFilesError, BAD_ZIP, include_regexes=['good\.nc'])
+        eligible_filenames = handler.file_collection.filter_by_attribute_id_not('publish_type',
+                                                                                PipelineFilePublishType.UNSET).get_attribute_list('name')
         self.assertIn('good.nc', eligible_filenames)
         self.assertNotIn('bad.nc', eligible_filenames)
```

Comment:
I know it's not part of this PR, but this would be a stronger assertion than the above two lines:
```python
self.assertListEqual(['good.nc'], eligible_filenames)
```
Same in `test_exclude` below.

Reply:
Those have been there since the start I reckon! But yes, it should be updated.